src/organisms/cards/GenericCard/Readme.md

```diff
@@ -2,10 +2,41 @@ Generic Card Example:
 
 ```example
     <GenericCard
-      header='Janis Ian'
-      subheader='Added to policy'
       onClose={() => alert('I feel so closed rn')}
       footerText='Edit'
       onFooterLinkClick={() => alert("Ughhh~! You've clicked me")}
-    />
+    >
+      <Text tag={'h4'} type={4}>Janis Ian</Text>
+      <Text tag={'h5'} type={6} color='neutral-5'>Added to policy.</Text>
+    </GenericCard>
+```
+
+```example
+    <GenericCard>
```

Comment:
the console is complaining about some required props that are missing here - `onClose` and `footerText` - although we can probably not require them?